contracts/OwnedRegistrar.sol

```diff
@@ -50,6 +89,15 @@ contract OwnedRegistrar is RBAC {
         emit RegistrarRemoved(id, registrar);
     }
 
+    function defaultResolver() external view returns (address) {
+        return resolver;
+    }
+
+    function resetResolver(address newResolver) public onlyRole("owner") {
```

Comment:
I would suggest calling this `setResolver`; reset implies setting it back to a previous value.

Reply:
Thanks for the suggestion.